src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,34 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# TODO: Move this to Makefile. how, when I shouldn't edit it from this repo?
+.PHONY: component-download-mondo.owl
+component-download-mondo.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I http://purl.obolibrary.org/obo/mondo.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+# TODO: Move this to Makefile. how, when I shouldn't edit it from this repo?
+$(COMPONENTSDIR)/mondo.owl: component-download-mondo.owl
+	if [ $(COMP) = true ] ; then if cmp -s $(TMPDIR)/component-download-mondo.owl.owl $@ ; then echo "Component identical."; else echo "Component is different, updating." && cp $(TMPDIR)/component-download-mondo.owl.owl $@; fi; fi
+
+# min-id: the next available Mondo ID
+slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/mirror_signature-mondo.tsv | slurp/
+	python $(SCRIPTSDIR)/migrate.py \
+	--ontology-path $(COMPONENTSDIR)/%.owl \
+	--sssom-map-path $(TMPDIR)/mondo.sssom.tsv \
+	--min-id 123000 \
+	--mondo-terms-path $(REPORTDIR)/mirror_signature-mondo.tsv \
+	--outpath $@
+
+slurp-%: slurp/%.tsv
```

Comment:
Any idea why `make slurp-omim` fails?: ``make: *** No rule to make target `slurp-omim'.  Stop.``

Reply:
This happens when make cannot figure out at least one dependency, transitively. "Figure out" means, there is at least one dependency somewhere in the depedency chain for which neither a file, nor a make target exists.. If you cant figure it out I can debug this for you, just say the word!

Reply:
"the word!". That is a tiny pain I have though; no "debugging" in `make`, so to speak; just troubleshooting.

What I would expect though is that the resolution would look like this:
goal `slurp-omim` (`slurp-%`) -> prerequisite `slurp/omim.tsv` (`slurp/%.tsv`) -> goal `slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/mirror_signature-mondo.tsv | slurp/`

But for some reason, this doesn't happen.

Reply:
Shall I take care of this?

Reply:
Yes, please.

Reply:
Ok, I didnt test it, but I fixed some issues I could see by sight in https://github.com/monarch-initiative/mondo-ingest/commit/72c3299b9b123097baa689d0cf9c4aa0b06988db

See commit message for details. Also read https://stackoverflow.com/questions/7887343/makefile-wildcard-static-rule-with-phony

Reply:
I read the commit message:

> 2. PHONY goals cannot have wildcard dependencies

I see! Now I understand why `slurp-omim` wasn't working!. Seems like poor language design UX, IMHO. Very good to know this.

@matentzn Marking this resolved. Just tagging you so you are aware that I understand.